guides/sdk-labels.md

```diff
@@ -0,0 +1,37 @@
+# SDK's issues Labels
+
+⚠️ These labels are not the same as this repo's labels. They are meant to be used to tag the issues opened in the different SDK's repositories.
+
+We have many issues in every integration repository. These help us understand what each issue is about or its state.
+A clear label prevents us from re-reading the issue when we are scanning them, and they provide a context in the blink of an eye.
+
+Nonetheless, until now, every repo used its own label system. Thus, to avoid adapting to every repo's label choices, we want to create a consistent labels list to use.
+
+These are the labels that are encouraged to use.
+
+## Default labels
+
+They already exist in every repo.
+
+| label Name  | Color  | description  |
+|---|---|---|
+|`bug` | #d73a4a | Something isn't working |
+| `good first issue` | #7057ff | Good for newcomers |
+| `help wanted` | #008672 | Extra attention is needed |
+| `enhancement` | #a2eeef | New feature or request |
+| `security` | #ee0701 | Pull request or issue that has recieve no activity for a long time. |
+
+## Custom labels
+
+They need to be added manually
+
+| label Name  | Color  | description |
+|---|---|---|
+| `support` | #F78FB2| Issues related to support questions. |
```

Comment:
Maybe we can add that they will be handled by the dev rel team?

Reply:
I think we don't need to be in line with them, if you find it a good idea you can add it, no hard feelings :D